src/main/resources/entity-types/finance/drv_ledger_details.json5

```diff
@@ -0,0 +1,340 @@
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'netTransfers'"
+    },
+    {
+      name: 'created_by_username',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'createdByUsername'"
+    },
+    {
+      name: 'updated_by_username',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'updatedByUsername'"
+    },
+    {
+      name: 'unavailable',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'unavailable'"
+    },
+    {
+      name: 'initial_allocation',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'initialAllocation'"
+    },
+    {
+      name: 'allocation_to',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'allocationTo'"
+    },
+    {
+      name: 'allocation_from',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'allocationFrom'"
+    },
+    {
+      name: 'total_funding',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'totalFunding'"
+    },
+    {
+      name: 'cash_balance',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'cashBalance'"
+    },
+    {
+      name: 'awaiting_payment',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'awaitingPayment'"
+    },
+    {
+      name: 'encumbered',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'encumbered'"
+    },
+    {
+      name: 'expenditures',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'expenditures'"
+    },
+    {
+      name: 'over_encumbrance',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'overEncumbrance'"
+    },
+    {
+      name: 'over_expended',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'overExpended'"
+    }
```

Comment:
Individual comments were getting kind of old. But all of these should be floats.